action.sh

```diff
@@ -64,8 +69,15 @@ echo "Getting hash for base repository commit"
 HASH=$(echo "${GITHUB_SHA}" | cut -c1-7)
 
 # Now add all the changes and commit and push
-git add . && \
-git commit -m "Auto publishing site from ${GITHUB_REPOSITORY}@${HASH}" && \
-git push --force "${TARGET_REPO_URL}" master:master
+if [[ "${TARGET_BRANCH}" = "master" ]]; then
+  git add . && \
+  git commit -m "Auto publishing site from ${GITHUB_REPOSITORY}@${HASH}" && \
+  git push --force "${TARGET_REPO_URL}" master:master
+else
+  git checkout -b ${TARGET_BRANCH}
```

Comment:
I have tested that without running this in the same command `&& \`, it would still be run

And for the sake of simplicity, amending the symbols

Reply:
I'd suggest only putting the checkout in the if statement. That means we don't need to then duplicate the next three lines. I think a git push to url and master will still work for both scenarios?

Reply:
I'd prefer to keep the `&&` please. It means if something goes wrong, it doesn't try to move onto the next command.

Reply:
Okay, got your point.
Have made the change